ui/apps/platform/src/Containers/ComplianceEnhanced/Schedules/ScanConfigActionDropdown.tsx

```diff
@@ -59,7 +61,7 @@ function ScanConfigActionDropdown({
             key="Edit scan schedule"
             component="button"
             // description={isScanning ? 'Edit is disabled while scan is running' : ''}
-            // isDisabled={isScanning}
+            isDisabled={isScanning || isReportStatusPending}
```

Comment:
What are pro and con of something like `isDisabled={isDisabled}` and

```ts
const isDisabled = isScanning || isReportStatusPending;
```

Reply:
Both ways work. What you described above would simplify the logic to reuse for different buttons. We can do that. The only downside I can think of would be if there is a specific case where we only need one of those and not both. I doubt that would be an edge case though.

I thought about it a bit more and I was thinking we could call it `isProcessing`. It sounds more correct, and for the case where we have 

```
isDisabled={notifiers.length === 0 || isProcessing}
```

It sounds more correct than

```
isDisabled={notifiers.length === 0 || isDisabled}
```